src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -209,6 +209,10 @@ private void CoreWebView2OnWebMessageReceived(object sender, CoreWebView2WebMess
         /// </summary>
         public void Dispose()
         {
+            if (initState == InitializeState.Started)
+            {
+                Log("DocumentationBrowserView is being disposed but async initialization is still not done");
```

Comment:
Will there be an issue due to this?

Reply:
My question is - shouldn't we protect against that or are you saying the chances of that happening are low?